lib/interface/cli/commands/workflow/get.cmd.js

```diff
@@ -65,12 +66,14 @@ const command = new Command({
                 const pipelines = await pipeline.getAll();
                 _.forEach(pipelineNames, (pipelineName) => {
                     const matchPipelines = _.find(pipelines, pipeline => pipeline.info.name.toString() === pipelineName);
-                    if (_.isArray(matchPipelines)) {
-                        _.forEach(matchPipelines, (currPipeline) => {
+                    if (matchPipelines) {
+                        const arrMatchPipelines = _.isArray(matchPipelines) ? matchPipelines : [matchPipelines];
+                        _.forEach(arrMatchPipelines, (currPipeline) => {
                             pipelineIds.push(currPipeline.info.id);
                         });
                     } else {
-                        pipelineIds.push(matchPipelines.info.id);
+                        printError('Cannot find any workflows with these pipelines names');
```

Comment:
please don't exit the process.
throw an error which will cause the process to exit